alica_tests/src/test/test_alica_tracing.cpp

```diff
@@ -52,47 +89,101 @@ TEST_F(AlicaTracingTest, runTracing)
     auto log = logs.front(); // MasterPlan
     EXPECT_EQ(log.first, "Plan");
     EXPECT_EQ(log.second, "true");
-    logs.pop();
+    logs.erase(logs.begin());
 
     log = logs.front(); // MasterPlan Init
     EXPECT_EQ(log.first, "Init");
     EXPECT_EQ(log.second, "true");
-    logs.pop();
+    logs.erase(logs.begin());
 
     log = logs.front(); // SubPlan
     EXPECT_EQ(log.first, "Plan");
     EXPECT_EQ(log.second, "true");
-    logs.pop();
+    logs.erase(logs.begin());
 
     log = logs.front(); // SubPlan Init
     EXPECT_EQ(log.first, "Init");
     EXPECT_EQ(log.second, "true");
-    logs.pop();
+    logs.erase(logs.begin());
 
     log = logs.front(); // Behaviour
     EXPECT_EQ(log.first, "Behaviour");
     EXPECT_EQ(log.second, "true");
-    logs.pop();
+    logs.erase(logs.begin());
 
     log = logs.front(); // Behaviour Init
     EXPECT_EQ(log.first, "Init");
     EXPECT_EQ(log.second, "true");
-    logs.pop();
+    logs.erase(logs.begin());
 
     log = logs.front(); // Behaviour Run
     EXPECT_EQ(log.first, "Run");
     EXPECT_EQ(log.second, "true");
-    logs.pop();
+    logs.erase(logs.begin());
 
     log = logs.front(); // Behaviour Terminate
     EXPECT_EQ(log.first, "Terminate");
     EXPECT_EQ(log.second, "true");
```

Comment:
@veerajsk  probably changed this. we are going to have status = init | run | terminate